dto/src/test/java/ca/on/oicr/gsi/runscanner/rs/dto/test/AbstractNotificationDtoTest.java

```diff
@@ -0,0 +1,54 @@
+package ca.on.oicr.gsi.runscanner.rs.dto.test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import ca.on.oicr.gsi.runscanner.dto.NotificationDto;
+import ca.on.oicr.gsi.runscanner.dto.type.HealthType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.util.ISO8601DateFormat;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.time.LocalDateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+public abstract class AbstractNotificationDtoTest {
+  NotificationDto notificationDto;
```

Comment:
Since doing this requires a lot of casting in the derived classes, can you make this a field in the children of the derived type and add an abstract method that returns `NotificationDto` that is used by these test?